.test-defs/TestSuiteShootDefaultSerial.yaml

```diff
@@ -22,5 +22,6 @@ spec:
       -fenced=$FENCED
       -ginkgo.focus="\[DEFAULT\].*\[SERIAL\]"
       -ginkgo.skip="\[DISRUPTIVE\]"
+      -ginkgo.timeout=3h
```

Comment:
I see that in line 8 we define `activeDeadlineSeconds: 7200` (2h) as the timeout for the whole test definition, hence testmachinery will kill this test after 2h. So either you could use here `2h` as well or prolong the testmachinery definition to i.e. `3h`, this could be useful if e.g. ginkgo would emit some interesting logs/dumps when it reaches its defined timeout and we thus avoid that testmachinery kills this before ginkgo could dump.

In the same line, probably it makes sense to go over all test-defs to see if there's a mismatch between the new ginkgo default and whats already defined as testmachinery timeout.

Reply:
I checked the times even for the slower providers again and I think we can lower the `ginkgo.timeout` here to 2 hours. Then this would also match the `activeDeadlineSeconds`. I will also have a look at the other test-defs.